src/components/course/tests/EnrollButton.test.jsx

```diff
@@ -18,23 +21,34 @@ import {
   ENROLL_BUTTON_LABEL_NOT_AVAILABLE,
 } from '../data/constants';
 
+jest.mock('../../dashboard/sidebar/offers', () => ({
+  fetchOffers: () => {},
+}));
+
+const mockStore = configureMockStore([thunk]);
+
 /* eslint-disable react/prop-types */
 const EnrollButtonWithContext = ({
   initialAppState = {},
   initialCourseState = {},
   initialUserSubsidyState = {},
+  initialReduxStore = {},
 }) => (
   <AppContext.Provider value={initialAppState}>
     <UserSubsidyContext.Provider value={initialUserSubsidyState}>
       <CourseContextProvider initialState={initialCourseState}>
-        <EnrollButton />
+        <reactRedux.Provider store={mockStore(initialReduxStore)}>
```

Comment:
I am a bit rusty now on this side of things but this looks interesting, to init the redux store in the test